### ISSUE TITLE
A11y_AzureExtensionsforVSCode_AzureFuction_DefaultScreen_ColorContrast : The "ms-azuretools.vscode-azurefunctions" text has color contrast ratio less than 4.5:1

### ISSUE THREAD

**Issue by nirmalsahoo** [microsoft/vscode#104551]
GitHubTags: #A11yMAS; #A11yTCS; #MAC; #ColorContrast; #A11y_VSCode; #MAS1.4.3; #A11ySev2; #AI4D;#CAITestPass_May2020;#Benchmark;#Visual Studio Code;#FTP;
**Environment Details:**
Application Name: Azure Extensions for VS Code
Application Version: 1.48.0-insider
Commit: cfbd1999769f4f08dce29629fb92fdc0fac53829
Date: 2020-08-06T05:40:40.871Z
Electron: 7.3.2
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.6.0
Operating system: macOS Catalina Version 10.15.6 (19G73)
Screen Reader: VoiceOver

**Repro Steps:**
Open Visual studio code
Click on extension button and search for "Azure Functions"
Click on install and move to the "Azure" button, the "Default Screen" will appear.
Navigate to "ms-azuretools.vscode-azurefunctions" text and verify if the color contrast ratio is less than 4.5:1 or not.
**Actual:**
The "ms-azuretools.vscode-azurefunctions" text has color contrast ratio of 2.6:1 which is less than the standard color contrast ratio of 4.5:1 which is not appropriate as the users with low vision will have difficulty in reading the text on the screen.

**Expected:**
The color contrast ratio of "ms-azuretools.vscode-azurefunctions" text should be greater than or equal to 4.5:1.

**User Impact:**
If the color contrast ratio of the content present on the screen is less than 4.5:1 then the users with low vision will have difficulty in reading the data on the screen.

**Attachment link for Reference:**

**Comment by isidorn** [comment]
Please file against ms-azuretools.vscode-azurefunctions extension.
Then that team can identify if it is really an issue in vscode and file it here with minimal human readable steps.

**Comment by nirmalsahoo** [comment]
@isidorn We have already logged bug against ms-azuretools.vscode-azurefunctions extension.

[https://github.com/microsoft/vscode-azurefunctions/issues/2280](url)

**Comment by isidorn** [comment]
Great, thanks
